string[] joint_name
uint8[] script_number
---
string result
